### DISCUSSION TITLE
TypeORM No metadata for "tableName" was found

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#16941] [Questions]
I'm trying to specify dataSource from where to get entities:
`/src/server/services/database-service.ts`
line: `entities: ['../entities/*.ts']`
```ts
import 'reflect-metadata';
import { DataSource } from 'typeorm';

const { DB_HOST, DB_USER, DB_PASSWORD, DB_NAME } = useRuntimeConfig();

export const AppDataSource = new DataSource({
	type: 'postgres',
	host: DB_HOST,
	port: 5432,
	username: DB_USER,
	password: DB_PASSWORD,
	database: DB_NAME,
	entities: ['../entities/*.ts'],
	synchronize: true,
	logging: false,
	migrationsTableName: 'migrations',
	migrations: ['/migrations'],
});

export async function connect() {
	if (!AppDataSource.isInitialized) {
		try {
			await AppDataSource.initialize();
		} catch (error) {
			throw error;
		}
	}
}

export async function close() {
	try {
		await AppDataSource.destroy();
	} catch (error) {
		throw error;
	}
}
```
but i get an error: `No metadata for "User" was found.` is my table in DB

**Comment by palamarchukser** [comment]
👋 @mihail727 looks like the entity path is wrong. Check that the path of the objects is defined according to the compiled dist/build folder and not the src folder.

try to use something like this
`entities: [__dirname + '/../**/*.entity.{js,ts}']`

**Reply by mihail727** [reply]
`__dirname` is undefined here(

**Reply by mihail727** [reply]
`[worker reload] [worker init] __dirname is not defined in ES module scope `
